Dangerfile

```diff
@@ -119,7 +119,7 @@ end
 
 ################################################
 # Check Documentation TOC update
-doc_dir_structure_modified = (git.added_files + git.deleted_files + git.renamed_files.map(&:values)).any? { |path| path.start_with?('Documentation/') }
+doc_dir_structure_modified = (git.added_files + git.deleted_files + git.renamed_files.map(&:values)).flatten.any? { |path| path.start_with?('Documentation/') }
```

Comment:
`git.renamed_files` is not an `Array` of `Strings` (like `added_files` and `deleted_files`), but an `Array` of `Hashes` (aka Dictionaries), each Hash having 2 keys, `before` and `after`.

So for each entry, calling `entry.values` returns a Array with 2 values; which means that `git.renamed_files.map(&:values)` returns an Array of 2-values-Arrays. Hence the `.flatten` needed here.